vault/raft.go

```diff
@@ -315,8 +422,14 @@ func (c *Core) startPeriodicRaftTLSRotate(ctx context.Context) error {
 	// checkCommitted verifies key updates have been applied to all nodes and
 	// finalizes the rotation by deleting the old keys and updating the raft
 	// backend.
-	checkCommitted := func() error {
-		keyring, err := readKeyring()
+	checkCommitted := func(haOnly bool) error {
+		// No-op here if we're using raft for HA-only. Since the storage is
+		// shared, the two phase commit is not done on rotation.
+		if haOnly {
```

Comment:
Since we split these into two functions is this still necessary?

Reply:
Good catch, no longer necessary!